components/CardList/index.js

```diff
@@ -5,13 +5,22 @@ import { useContext } from "react";
 import LoadingSpinner from "../LoadingSpinner";
 import FilterSection from "../FilterButtonSection";
 import { filterButtonsData } from "../../utils/data/filterButtonsData";
+import { useState } from "react";
+import { useEffect } from "react";
+import SearchField from "../SearchField";
 
 export default function CardList() {
   const breedData = useContext(BreedData);
-  const activeButtonId = useContext(Filter);
+  const { activeButtonId, setActiveButtonId } = useContext(Filter);
+
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    setActiveButtonId({ id: 1 });
```

Comment:
Why are you using an object `{ id: 1 }` inside of a state that is already called `Id`?

I'd suggest something like this: `setActiveButtonId(1);`